pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -1727,6 +1727,14 @@ func parseConfigItems(config *zconfig.EdgeDevConfig, ctx *getconfigContext) {
 	//  value or retain the previous value.
 	gcPtr := &ctx.zedagentCtx.globalConfig
 	newGlobalConfig := types.DefaultConfigItemValueMap()
+	// Note: UsbAccess is special in that it has two defaults.
+	// When the device first boots the default is "true" as specified
+	// in the DefaultConfigItemValueMap. But when connecting to the
+	// controller, if the controller does not include the item, it
+	// should default to "false".
+	// That way bringup of new hardware models can be done using an
+	// attached keyboard.
+	newGlobalConfig.SetGlobalValueBool(types.UsbAccess, false)
```

Comment:
Missed this.. This doesn't sound like the right way?? 
I guess the behavior we want is:

1) Before Device Registration, USB access should always be True
2) After registration - and from that point on - USB Access should come from the controller.

So here, the default specified in the DefaultConfigItemValueMap should be false - but based on the state of the device, we should over-ride it,.